packages/core/upload/admin/src/components/FiltersPicker/FiltersCard/init.js

```diff
@@ -1,21 +1,30 @@
 const init = (initialState, timestamps) => {
   const [created_at, updated_at] = timestamps;
 
-  return initialState
-    .update('name', () => created_at)
-    .updateIn(['filtersForm'], object => {
-      return object.keySeq().reduce((acc, current) => {
-        if (current === 'created_at' && created_at !== 'created_at') {
-          return acc.set(created_at, object.get('created_at')).remove('created_at');
-        }
-
-        if (current === 'updated_at' && updated_at !== 'updated_at') {
-          return acc.set(updated_at, object.get('updated_at')).remove('updated_at');
-        }
-
-        return acc;
-      }, object);
-    });
+  const filtersForm = Object.keys(initialState.filtersForm).reduce((acc, current) => {
+    // The timestamps can be customised so we need to update them
+    if (current === 'created_at') {
+      acc[created_at] = initialState.filtersForm.created_at;
+
+      return acc;
+    }
+
+    if (current === 'updated_at') {
+      acc[updated_at] = initialState.filtersForm.updated_at;
+
+      return acc;
+    }
+
+    acc[current] = initialState.filtersForm[current];
+
+    return acc;
+  }, {});
```

Comment:
Isn't this the same as:

```js
const filtersForm = {
 ...initialState.filtersForm,
 created_at: initialState.filtersForm.created_at,
 updated_at: initialState.filtersForm.updated_at
}
```
???

Reply:
Not exactly, in case we have custom timestamps we remove the create_at and updated_at keys.

Reply:
More code will be added? or am I missing smth?

Reply:
ok sorry, I didn't see the line outside the scope of the modif